astropy/units/tests/test_units.py

```diff
@@ -171,7 +171,7 @@ def test_unknown_unit3():
     assert unit == "FOO"
     assert unit != u.m
     # next two from gh-7603.
-    assert unit != None  # noqa
+    assert unit is not None
```

Comment:
Why not just slap the `# noqa: E711` to be consistent with the test below?

Reply:
Fixed in 757d1d9fd8c8b55ade21f6dc2d566c9ca5e8047f